orchagent/bfdorch.cpp

```diff
@@ -74,6 +74,16 @@ BfdOrch::BfdOrch(DBConnector *db, string tableName, TableConnector stateDbBfdSes
 
     Orch::addExecutor(bfdStateNotificatier);
     register_state_change_notif = false;
+    BgpGlobalStateOrch* bgp_global_state_orch = gDirectory.get<BgpGlobalStateOrch*>();
```

Comment:
I think you don't need this section. Just initialize `tsa_enabled `to false in class and handle TSA change in the normal update path.

Reply:
fixed